### DISCUSSION TITLE
Compatibility with OCCP

### DISCUSSION THREAD

**Discussion by edwinvandenbelt** [AtlasPublicPolicy/charging-use-spec#6] [Ideas]
Actually, this is not an issue, but how does this initiative relate to OCPP? https://www.openchargealliance.org/protocols/ocpp-201/

**Comment by jamesdifilippo** [comment]
Great question! OCCP is an operational and communications protocol that defines the interface between networked charging hardware (EVSE) and a centralized system. It is meant to provide a standardized communication layer between EVSE and backend systems (for billing, management, smart charging, etc) typically hosted by a EV service provider (network). 

In contrast, the Charging Use Specification comes in downstream of communication between the chargers and the central system. It provides a standardized way for the centralized system that records individual charging data (from networked chargers) to communicate that usage data for compliance purposes. In addition to standardizing reporting data, the Charging Use Specification defines a data hierarchy for linking session data to chargers, physical locations, and groups of chargers funded under a single funding award.

The spec relates to OCCP in that the reporting data is initially generated by the EVSE and so data available to report in this spec is dependent on OCCP (or a proprietary communications protocol) supporting communication of that data. Given that both OCCP and this specification are open source It is possible to write software that would translate raw OCCP data into the session data reporting format specified here.
